katportalclient/test/test_client.py

```diff
@@ -1556,7 +1636,6 @@ def fake_http_response(response_string):
     future.set_result(result)
     return [future]
 
-
```

Comment:
This line should not have been removed.  pep8 requires two lines between functions.  One between methods.